c/src/ml-api-inference-single.c

```diff
@@ -1111,6 +1155,132 @@ ml_single_open_full (ml_single_h * single, const char *model,
   return ml_single_open_custom (single, &info);
 }
 
+/**
+ * @brief`Creates an option and returns the instance a handle.
+ */
+int
+ml_option_create (ml_option_h *option) {
+  ml_option *_option;
+
```

Comment:
In all functions about ml-option, add `check_feature_state (ML_FEATURE);` and update header (return code, @retval #ML_ERROR_NOT_SUPPORTED Not supported.).